### DISCUSSION TITLE
https://github.com/FlareSolverr/FlareSolverr/issues/1335 issue got resolved without proper linked issue

### DISCUSSION THREAD

**Discussion by khalith** [FlareSolverr/FlareSolverr#1342] [Q&A]
We have been facing issue for flaresolverr and given the relevant logs to get some help with respect to application but got closed without any relevant issues / solution.

**Comment by ilike2burnthing** [comment]
> ### Is there already an issue for your problem?
> - [X] I have checked older issues, open and closed

Try again.
